### DISCUSSION TITLE
Git storage operates in wrong directory

### DISCUSSION THREAD

**Discussion by madduck** [requarks/wiki#3674] [Error / Bug Report]
**Describe the bug**

I've configured Git storage to use `./data/repo`. However, writes fail, I get an error, and it seems as if the server is trying to create the Git repository in `.`, which the user running the server cannot write to:

```
May  5 10:33:48 arnold node[232662]: 2020-05-05T08:33:48.036Z [MASTER] #033[32minfo#033[39m: (STORAGE/GIT) Committing new file [en] home...
May  5 10:33:48 arnold node[232662]: 2020-05-05T08:33:48.056Z [MASTER] #033[33mwarn#033[39m: fatal: Unable to create '/srv/apache2/wiki.edsfellas.nz/wiki/.git/index.lock': Permission denied
```

**To Reproduce**

1. Any write to the database.

**Expected behavior**

I expect the page to be saved to the local Git repo, stored under `./data/repo`, as configured.

**Host Info (please complete the following information):**
 - OS: Debian 10
 - Wiki.js version: 2.3.81
 - Database engine: PostgreSQL 11.7

**Comment by Smankusors** [comment]
can you use absolute path instead of relative path?

**Comment by madduck** [comment]
@Smankusors I just tried `../repo` and that actually works fine. I assume the absolute path will also work. Want me to explicitly try?

**Comment by NGPixel** [comment]
Have you git cloned the wiki project by any chance? Or you installed by downloading the tar.gz? You would get that error if there was already a git folder in a parent folder.

**Comment by madduck** [comment]
Tarball, and there is no Git directory far or near.
